### PR TITLE
Hotfix np-46386 with organization

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search2/ticket/Constants.java [hunk 1]
```diff
@@ -40,152 +41,70 @@ public final class Constants {
...
-                                                  + DOT
-                                                  + LAST_NAME
-                                                  + DOT
-                                                  + KEYWORD
-                                                  + PIPE
-                                                  + VIEWED_BY
-                                                  + DOT
-                                                  + USERNAME
-                                                  + DOT
-                                                  + KEYWORD;
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search2/ticket/Constants.java:114]: Auto format fubar

File: search-commons/src/main/java/no/unit/nva/search2/ticket/Constants.java [hunk 2]
```diff
@@ -40,152 +41,70 @@ public final class Constants {
...
-                                                     + DOT
-                                                     + LAST_NAME
-                                                     + DOT
-                                                     + KEYWORD
-                                                     + PIPE
-                                                     + FINALIZED_BY
-                                                     + DOT
-                                                     + USERNAME
-                                                     + DOT
-                                                     + KEYWORD;
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search2/ticket/Constants.java:178]: more fubar

File: search-commons/src/test/resources/datasource_urls_ticket.json [hunk 3]
```diff
@@ -1,36 +1,36 @@
 {
-  "page=0&size=21": 21,
+  "page=0&size=21": 19,
```
Comment (anetteOlli) [inline search-commons/src/test/resources/datasource_urls_ticket.json:2]: Each time you change something; this needs to be changed as well?

I can see the pain and suffering from this file for you guys that work with this.

**Review by anetteOlli — APPROVED** [review]

File: search-commons/src/test/resources/datasource_urls_ticket.json (hunk 3 above)
Comment (StigNorland) [reply search-commons/src/test/resources/datasource_urls_ticket.json:2]: I needed to test a filter , and added it, filtering away two documents, som tests didn't include it some did.
What I gain is a real life "database" with consequences for changing data and/or logic, as in the real world.

**Pull request merged**